addons/frag/functions/fnc_addBlackList.sqf

```diff
@@ -1,4 +0,0 @@
-#include "script_component.hpp"
-private ["_round"];
-_round = _this select 0;
-GVAR(blackList) set[(count GVAR(blackList)), _round];
```

Comment:
Are we removing the blacklist? This function might actually be useful to keep around. Though we do need to adjust it to a pushback.